deploy/internal/deployment-endpoint.yaml

```
@@ -21,6 +19,8 @@ spec:
       labels:
         noobaa-s3: noobaa
         app: noobaa
+      annotations:
+        noobaa.io/configMapHash: ""
```

just to be more consistent with conventions 
```suggestion
        noobaa.io/configmap-hash: ""
```